.releaserc.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
     branches: [
         {name: 'main'},
         {name: '1.87.x', range: '1.87.x', channel: '1.87.x'},
+        {name: 'v2-saml', prerelease: 'beta'},
```

Comment:
just a reminder to remove this line as soon as we don't need a new prerelease anymore